test/fixtures/spec/shr_fhir.txt

```diff
@@ -3,8 +3,9 @@ Namespace:  shr.fhir
 
 // contains sample elements that exercise FHIR mappings
 
-EntryElement: PatientSingleConstraintEntry
-Description:	"Maps to the Patient resource with one cardinality-constrained field"
+EntryElement: PatientConstraintsEntry
+Description:	"Maps to the Patient resource with constraints"
+Value:        1..1 date
```

Comment:
I think this should be `Value: date` rather than `Value: 1..1 date`. I don't think you can constrain values; they're just inherently 1 element